src/styles/blocks/button.scss

```
@@ -0,0 +1,24 @@
+.button {
+  @include padding-inline;
+
+  display: inline-block;
+  width: 166px;
```

Need to fix it everywhere
```suggestion
  width: 100%;
```